tips/views.py

```diff
@@ -1,7 +1,8 @@
 from django.shortcuts import render, redirect, get_object_or_404
 from .models import Tip
 from .forms import TipForm
-from .constants import TIPS_PAGE_ROUTE, ADD_TIP_PAGE_ROUTE, EDIT_TIP_PAGE_ROUTE
+from .constants import TIPS_PAGE_ROUTE, ADD_TIP_PAGE_ROUTE, EDIT_TIP_PAGE_ROUTE, READ_MORE_PAGE_ROUTE,\
```

Comment:
Is the '\' at the end necessary?

Reply:
Yes, without it the flake 8 returns error it. It is for continue the code in new line

Reply:
ok 👍